.env-template

```diff
@@ -27,4 +27,8 @@ AZURE_CHAT_ENDPOINT_URL=<ENDPOINT URL>
 USE_EXPERIMENTAL_LANGCHAIN=False
 
 # Community features
-USE_COMMUNITY_FEATURES='True'
\ No newline at end of file
+USE_COMMUNITY_FEATURES='True'
+
+# searching elastic search cluster
+ELASTICSEARCH_HOST=
```

Comment:
Nit: Can you move this under the `#TOOLS` section? And change the comment to `#Searching Elasticsearch cluster`